front-end/src/Components/EditProfile.js

```diff
@@ -1,12 +1,31 @@
 import { useState, useEffect } from 'react'
 import { useParams, useHistory } from 'react-router-dom'
 import { firestoreThisImage } from '../Util/imageStore'
+import { gender, experience, radius, goals } from '../Util/searchFields'
```

Comment:
We had a file containing all the value, label for each dropdown,  src/Util/searchFields.js